### ISSUE TITLE
undefined: Asset

### ISSUE THREAD

**Issue by jwineinger** [cristim/autospotting#35]
I'm trying to test this out but I'm having trouble even getting it running locally.  Following the SETUP.md, I get this:

```
$ go get github.com/cristim/autospotting
# github.com/cristim/autospotting
src/github.com/cristim/autospotting/autospotting.go:53: undefined: Asset
src/github.com/cristim/autospotting/autospotting.go:58: undefined: Asset
```

**Comment by cristim** [comment]
@jwineinger thanks for reporting this, good catch!

Here's a bit of background: I recently refactored the project to use [eawsy/aws-lambda-go](https://github.com/eawsy/aws-lambda-go) and it more or less required [jteeuwen/go-bindata](https://github.com/jteeuwen/go-bindata), so in the process it seems I broke the support for `go get`.

If you use `git clone` as usual and then run `make` inside the directory, it will fetch/generate the required data for those static assets and will use the `go-bindata` tool to generate a bunch of source files that provide that Asset function.

Even though I wouldn't normally check in such generated code, I think in this case it makes sense to do it, in order to be able to build the project with `go get` just like any other golang program.

I will commit a fix shortly.
